.github/CODEOWNERS

```diff
@@ -668,7 +668,7 @@
 # ServiceOwners:                                                   @bastionsuppgithub
 
 # ServiceLabel: %Network - CDN
-# ServiceOwners:                                                   @cdnfdsuppgithub
+# ServiceOwners:                                                   @cdnfdsuppgithub @ptnan7 @t-bzhan
```

Comment:
If these are for the management library, we'll want to move it down to the management section near the bottom of the file and add "%Mgmt" to the label list.   Otherwise, this will only apply to data-plane libraries as the catch-all will triage to Arthur.  

Please see the giant comment block at the beginning of the management section for details.